templates/components/starRating.build.hbs

```diff
@@ -1,7 +1,9 @@
-<template v-for="(option, index) in values">
-  <input class="rating-input" :name="name" type="radio" :id="name + '-' + index" v-model="value" :value="option.id" v-on:change="updateValue()">
-  <label class="rating-star" :for="name + '-' + index">
-    <i class="fa fa-star-o"></i>
-    <i class="fa fa-star"></i>
-  </label>
-</template>
+<div class="inverse-direction">
```

Comment:
indentation is incorrect here

Reply:
@squallstar That's how I saved it: https://cl.ly/0u1b3S1S1k0y